src/tests/network-no-diff.test

```diff
@@ -36,7 +36,7 @@ post()
     runtime=$(sed -E -n 's/.*@@@ runtime: (.*) @@@.*/\1/p' "$(basename $0).d/baseline.tmux.log")
 
     # If this system can't actually report runtime, bail.
-    if [ "$runtime" -eq "-" ]; then
+    if [ -z "$runtime" -o "$runtime" -eq "-" ]; then
```

Comment:
Done-- not a big issue for test/build code, but still worth fixing here, in your shell-test PR, and the new tests I just committed.